src/System.Private.CoreLib/shared/System/ReadOnlySpan.cs

```diff
@@ -40,7 +40,8 @@ public bool IsEmpty
             [NonVersionable]
```

Comment:
cc @jkotas, can we make such changes even if this property is marked as `NonVersionable`?

Reply:
This is ok. It is not changing behavior (for valid lengths). We have done the same change for IsNullOrEmpty that is also marked NonVersionable.